src/cf_warning/report.py

```diff
@@ -86,6 +88,7 @@ async def run_async(self):
 
         for level, df in data_segmented.items():
             table = df.reset_index(drop=True).to_markdown()
+            print(table)
```

Comment:
I forgot to delete this line. I was trying to print the information at first. Just for checking the table generated.